docker-compose/rails_client_bfd_v2.patch

```diff
@@ -0,0 +1,238 @@
+diff --git a/Gemfile b/Gemfile
```

Comment:
@JFU-GIT In my opinion, the rails client should not be patched in this repo. I think we should update the client for V2 capability in the bluebutton-sample-client-rails repository instead?

Reply:
agreed, this is temporary, to assist testing, will remove after rails clients support v2

Reply:
Agreed, some of the patch seems to be windows specific and would cause issues on other platforms.